client/src/components/CreatePost.test.tsx

```diff
@@ -25,34 +25,46 @@ afterEach(() => {
   }
 });
 
-describe("verifying launch of create post component", () => {
-  it("shows the preview button as disabled", () => {
+describe('verifying launch of create post component', () => {
+  it('shows the preview button as disabled', () => {
     // get the preview button
-    expect(screen.getByTestId("previewButton")).toBeDisabled();
+    expect(screen.getByTestId('previewButton')).toBeDisabled();
   });
 
-  it("should enable the preview button after providing required fields", () => {
+  it('should enable the preview button after providing required fields', () => {
     // input title
-    const titleTextField = screen.getByPlaceholderText("title");
-    fireEvent.change(titleTextField, { target: { value: "Test Club" } });
+    const titleTextField = screen.getByPlaceholderText('title');
+    fireEvent.change(titleTextField, { target: { value: 'Test Club' } });
     // input body
-    const bodyTextField = screen.getByPlaceholderText("description");
-    fireEvent.change(bodyTextField, { target: { value: "body" } });
+    const bodyTextField = screen.getByPlaceholderText(
+      'Description (minimum 25 characters)'
```

Comment:
The placeholder text tends to change often. I would use something like test id, id or some other attribute here.

goes for other uses of `getByPlaceholderText` as well

Reply:
I tried doing it with test id however it kept failing.
Turns out its because `TextField` is an abstraction by Material ui which was causing the issue.